### DISCUSSION TITLE
2.4 Updates

### DISCUSSION THREAD

**Discussion by jjm888** [tillsteinbach/VWsFriend#301] [Q&A] [answered]
With all the updates happening just now before the big move to 3.0 thought I would ask, check, as dealer is having issues updating my car. Been nearly 2 weeks. We pull the data from VW Servers but does the server trigger the car to give an update based on refresh time set?

**Comment by tillsteinbach** [comment, marked as answer]
No, the car is sending updates to the VW backend. VWsFriend is fetching the data from there. So VWsFriend is not triggering the car in any way. Only if you set e.g. climate from HomeKit the car is waking up.
So the update problems will not be caused by VWsFriend.
Do you maybe have a retrofit reverse camera? I heard that retrofitted parts can cause problems.

**Reply by jjm888** [reply]
That was my understanding. No reversing camera came as standard and yes some folks had issues. Keep up the good work. Thanks
